py_mmd_tools/mmd_elements.yaml

```diff
@@ -516,8 +732,10 @@ dataset_citation:
     acdd:
       publisher_name:
         separator: ','
-        comment: Required
+        comment: Required if not hosted by MET. If not an organisation add publisher_institution which is used to identify the data centre hosting the dataset. If multiple are listed, use comma separated list and consistency across fields.
```

Comment:
```suggestion
        comment: Required. If this is not an organisation, add publisher_institution to identify the data centre hosting the dataset. In case of multiple publishers, use comma separated list and consistency across fields.
```